### PR TITLE
Xenoarchaeology Cells Fix and Minor Tweaks

### PR DESCRIPTION
**Pull request by SleepyGemmy**
this PR fixes the xenoarchaeology cells being mislabelled and makes some minor visual tweaks.

### CODE REVIEW

**Comment by SleepyGemmy** [comment]
!review

**Comment by DreamySkrell** [comment]
image

Why remove the exterior catwalks and lattices? I think they look nice, otherwise it's just bland and featureless.

File: html/changelogs/SleepyGemmy-xenoarch_cells.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+author: SleepyGemmy
+
+delete-after: True
+
+changes:
+  - bugfix: "Fixed the xenoarchaeology cells being mislabelled."
```
Comment (FluffyGhoster) [inline html/changelogs/SleepyGemmy-xenoarch_cells.yml:6]: If you remove the exterior catwalks, the changelog should also reflect that

**Review by Alberyk — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Comment by Arrow768** [comment]
You need to make sure the current UT suite is ran against that PR.

**Comment by NonQueueingMatt** [comment]
reopen when the UTs are fixed

**Pull request closed without merging**